.github/workflows/weekly-featurenets-short-session-full-sha.yml

```diff
@@ -24,7 +24,7 @@ jobs:
     secrets: inherit
     with:
       featurenet-name: 'ops-test-full-hash'
-      ref: '787c39186884aaf86ed48f46f575a8cd8cfd13e8'
+      ref: '87ebe7c4856b4f3365ddba55b55089713a96631a'
```

Comment:
these sha are very opaque :{ how do you choose one?

Reply:
This is just a sha of `r-13-3`. Agree is not intuitive what should be there, but we'll do a follow-up step that returns a tag/SHA from the current Testnet - in the next PR.